buildSrc/src/main/kotlin/elastic-otel.java-conventions.gradle.kts

```diff
@@ -1,12 +1,50 @@
 plugins {
-  id("java")
+  java
+  id("elastic-otel.spotless-conventions")
```

Comment:
Why do we need to have spotless conventions in a separate file ? Is it to allow applying it both for the conventions themselves in addition to the project modules ?

Reply:
We could use spotless for non-java projects, though I don't think we'd ever do that. I did this more for a separation of concerns. If you prefer to have it directly in the java-conventions, I can put it in. I don't have a strong opinion here.

Reply:
Me neither, let's keep it this way then as it's usually easier to merge than to split later :-).